src/main/java/graphql/execution/ExecutionStrategy.java

```diff
@@ -588,4 +581,16 @@ protected void assertNonNullFieldPrecondition(NonNullableFieldWasNullException e
             throw e;
         }
     }
+
```

Comment:
This is painfully missing in Java :(

The ability to wait for a Collection<CF> support should be present